ldes-server-infra-postgres/postgres-ingest-repository/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/ingest/postgres/MemberPostgresRepository.java

```diff
@@ -54,12 +56,30 @@ INSERT INTO members (subject, collection_id, version_of, timestamp, transaction_
 							modelConverter.convertToDatabaseColumn(member.getModel()),
 					})
 					.toList();
-			return Arrays.stream(jdbcTemplate.batchUpdate(sql, batchArgs)).sum();
+
+			jdbcTemplate.batchUpdate(INSERT_SQL, batchArgs);
+
+			updateCollectionStats(members.size(), collectionId);
+
+			return members.size();
 		} catch (DuplicateKeyException e) {
 			return 0;
 		}
 	}
 
+	private void updateCollectionStats(int memberCount, int collectionId) {
+		final String SQL = """
```

Comment:
This should be `sql` or it should be a static final

Reply:
Uppercasing names is because of the `final` , not static. So this is correct